### ISSUE TITLE
Need graph generation example like GraphRNN, GraphVAE

### ISSUE THREAD

**Issue by yippp** [rusty1s/pytorch_geometric#826]
## ❓ Questions & Help
I have viewed the example codes in `example/`, the examples for graph autoencoders are about edges  generation for link prediction. It seems that there is no example for nodes generation.
I have no idea how to implement both nodes and edges generation network like molecule generation. Could you provide some relative exmaple, such as implementation of [GraphRNN](https://arxiv.org/pdf/1802.08773.pdf) and [GraphVAE](https://arxiv.org/pdf/1802.03480.pdf)?
Also, I will be pleasure if you could give me some hint using `torch_geometric` to implement them.

**Comment by rusty1s** [comment]
You are right that we currently do not provide any graph generation models. That is a current limitation.
The encoder-decoder scheme of the GraphVAE should be quite easy to implement with the help of PyTorch Geometric. In the end, this is just a stacking of GNN layers, global pooling, and MLPs. The hard part is to implement its reconstruction loss where you need to find the binary assignment matrix `X` first.

**Comment by yippp** [comment]
Thanks for your hint. 
Hope you can consider to add example for GraphRNN or GraphVAE. Nodes generation is really important for graph generation task. I think an example will inspire people to develop related model much easier.
